react-native-gutenberg-bridge/index.js

```diff
@@ -7,6 +7,12 @@ const isIOS = Platform.OS === 'ios';
 
 const gutenbergBridgeEvents = new NativeEventEmitter( RNReactNativeGutenbergBridge );
 
+// Leverages existing console polyfill from react-native which is a global function called
+// `nativeLoggingHook` that takes two parameters, a message and a log level.
+global.nativeLoggingHook = function( message, logLevel ) {
```

Comment:
I have a couple suggestions here:

* I think it'd be better to keep all `global` replacements in `src/globals.js` to make it easier to find in the future. The bridge could just expose a method that returns the logging hook.
* The apps should not rely on magic numbers. We should either convert logLevel to a string, or expose some named constants for all platforms in the bridge to make it easier to know what each value means.

Reply:
@koke it's not really magic, but part of `react-native`: https://github.com/facebook/react-native/blob/master/Libraries/polyfills/console.js#L376

Reply:
Makes sense! I'll leave soon so let me address after a while 🙇

Reply:
> @koke it's not really magic, but part of `react-native`

Oh, I know. I meant that from the apps perspective, there's no named constant exposed, so if you look at [how it's being handled](https://github.com/wordpress-mobile/gutenberg-mobile/blob/6b6e91125e3b375b20098c64ea049ee66746aa69/ios/gutenberg/GutenbergViewController.swift#L117), it's just mapping numbers to specific log levels in the apps, and it's not obvious there what possible values there are.

By [magic number](https://en.wikipedia.org/wiki/Magic_number_%28programming%29) I just meant an "unnamed numerical constant" 😄

Reply:
Yea we can have enum definition on Java and Swift side for sure.

Reply:
Addressed 🙇 99e4314...dcf1213